pkgs/data/themes/pop-gtk/default.nix

```
@@ -0,0 +1,62 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, sassc
+, gtk3
+, inkscape
+, optipng
+, gtk-engine-murrine
+, gdk-pixbuf
+, librsvg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pop-gtk-theme";
+  version = "2019-11-13";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "gtk-theme";
+    rev = "8ddf0fcad13c6a3f4fab2772b45561d74a8b4cce";
+    sha256 = "0byzxgd52291vsvwydafnq0xmnnggaryr9c9hfq5hxrd0rr08ibs";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    sassc
+    gtk3
+    inkscape
+    optipng
+  ];
+
+  buildInputs = [
+    gdk-pixbuf
+    librsvg
+  ];
+
+  propagatedUserEnvPkgs = [
+    gtk-engine-murrine
+  ];
+
+  postPatch = ''
+    for file in $(find -name render-\*.sh); do
+      patchShebangs $file
```

```suggestion
      patchShebangs "$file"
```

rather important in this case :smile:

---

Indeed, fixed it